### PR TITLE
Add some basic CI

### PR DESCRIPTION
**Pull request by micgro42**

### CODE REVIEW

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+jobs:
+    js-test:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v2
+
+            - name: Setup Node.js
+              uses: actions/setup-node@v2.5.1
+              with:
+                  node-version: "16.x"
```
Comment (lucaswerkmeister) [inline .github/workflows/test.yaml:13]: (I’d prefer `"16"` so it’s the same string as in `.nvmrc` but that’s a very minor nitpick)

**Review by lucaswerkmeister — APPROVED** [review]

File: .github/workflows/test.yaml (hunk 1 above)
Comment (micgro42) [reply .github/workflows/test.yaml:13]: but that makes no difference in terms of which version will be chosen, right?

Comment (micgro42) [reply .github/workflows/test.yaml]: done

Comment (lucaswerkmeister) [reply .github/workflows/test.yaml]: Yeah, should be purely cosmetic.

**Pull request merged**